cf/mixin/propertiesdatabounds.py

```diff
@@ -2229,16 +2238,19 @@ def override_calendar(self, calendar, inplace=False, i=False):
     def override_units(self, units, inplace=False, i=False):
         """Override the units.
 
-        The new units need not be equivalent to the original ones, and the
-        data array elements will not be changed to reflect the new
-        units. Therefore, this method should only be used when it is known
-        that the data array values are correct but the units have
-        incorrectly encoded.
+        The new units need not be equivalent to the original ones, and
+        the data array elements will not be changed to reflect the new
+        units. Therefore, this method should only be used when it is
+        known that the data array values are correct but the units
+        have incorrectly encoded.
 
-        Not to be confused with setting the `units` or `Units` attribute
-        to units which are equivalent to the original units.
+        Not to be confused with changing to equivalent units with the
+        `to_units` method or the `Units`, `units`, or `calendar`
+        attributes. These approaches also convert the data to have the
+        new units.
```

Comment:
I suggest being a bit more explicit with this sentence, else the crux of it could easily be missed. Something like:

```suggestion
        attributes. These approaches also convert the data values to conform
        with the new units.
```